prime-router/metadata/schemas/direct/imagemover-covid-19.schema

```diff
@@ -12,5 +12,10 @@ elements:
 
   - name: processing_mode_code
     default: P
-
+  - name: filler_order_id
```

Comment:
Add a space between fields as it is easier to read and follows other examples

Reply:
Agree.